osu.Game/Tests/Visual/Spectator/TestSpectatorClient.cs

```diff
@@ -134,7 +134,7 @@ protected override Task SendFramesInternal(FrameDataBundle bundle)
             FrameSendAttempts++;
 
             if (ShouldFailSendingFrames)
-                return Task.FromException(new InvalidOperationException());
+                return Task.FromException(new InvalidOperationException($"Intentional fail via {ShouldFailSendingFrames}"));
```

Comment:
This compiles, but looks wrong.

```suggestion
                return Task.FromException(new InvalidOperationException($"Intentional fail via {nameof(ShouldFailSendingFrames)}"));
```